bigquery.tf

```diff
@@ -3,6 +3,25 @@ locals {
   bigquery_dataset_name         = coalesce(var.bigquery_dataset_name, local.default_bigquery_dataset_name)
 }
 
+resource "google_bigquery_connection" "bq_spark_connection" {
+  connection_id = "bq-spark-conn-${lower(var.organisation_id)}-${lower(var.country_code)}"
+  location      = var.storage_location
+  description   = "BQ spark connection for ML"
+  spark {
+
+  }
```

Comment:
This block is not supported what it seems. Check here: https://github.com/LiveRamp/infrastructure/pull/36341

Reply:
Perhaps is this what's needed instead of spark?
```
cloud_resource {}
```
For more detailed documentation please check here: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_connection

Reply:
You are right, replaced spark with cloud_resource